product_docs/docs/pgd/4/harp/09_consensus-layer.mdx

```diff
@@ -149,49 +149,47 @@ zones in regions over traditional data centers.
 
 The process of selecting a leader begins with the startup of the first HARP manager:
 
-* Starts Postgres on the node
-* Reads the Postgres status (including any data on replication lag)
-* Queries the current leader (there should be none)
-* Stores the leadership request in the DCS
-* If the leadership request in not rejected, store the leader lease in the DCS
-* Refresh the leader lease (on a configurable interval, default is 500ms)
+* Starts Postgres on the node.
+* Reads the Postgres status, including any data on replication lag.
+* Queries the current leader. (There isn't one.)
+* Stores the leadership request in the DCS.
+* If the leadership request isn't rejected, stores the leader lease in the DCS.
+* Refreshes the leader lease. On a configurable interval, the default is 500ms.
 
-Once established, subsequent HARP manager instances startup in a similar fashion:
+Once established, subsequent HARP manager instances start up in a similar fashion:
 
-* Start Postgres on the node
-* Reads the Postgres status (including any data on replication lag)
-* Queries the current leader (there should be one existing)
-* As there is a leader, assume a shadow role and monitor the leader lease
-* If the lease expires, follow the path of the first HARP manager instance to claim leadership
+* Starts Postgres on the node.
+* Reads the Postgres status, including any data on replication lag.
+* Queries the current leader. (One now exists.)
```

Comment:
```suggestion
* Queries the current leader. (One should already exist for this to become a follower.)
```

Reply:
Is this edit okay? It's clearer to me now and I also think we don't need it to be a parenthetical now that I'm looking at it again.